data/json/items/tool_armor.json

```diff
@@ -1152,7 +1152,7 @@
             "draw_cost": 3,
             "max_volume": 4,
             "flags": [ "SHEATH_KNIFE", "SHEATH_SWORD" ]
```

Comment:
You should move this line to the outer scope and that will also fix the other issue